csrankings-m.csv

```diff
@@ -714,6 +715,7 @@ Marivic S. Tangkeko,De La Salle University,https://www.dlsu.edu.ph/colleges/ccs/
 Mariya Zheleva,University at Albany - SUNY,https://www.albany.edu/ceas/mariya-zheleva.php,eEqTPcwAAAAJ
 Marjan Sirjani,Mälardalen University,https://marjansirjani.github.io/Marjan-Sirjani/,TKnSrTQAAAAJ
 Marjo Kauppinen,Aalto University,https://people.aalto.fi/new/marjo.kauppinen,Wo8x6EUAAAAJ
+Mark A. Davenport,Georgia Insitute of Technology, https://mdav.ece.gatech.edu/index.html,DpuKx_oAAAAJ

```

Comment:
There is a typo in the institution's name. It should be "Georgia **Institute** of Technology" instead of "Georgia **Insitute** of Technology"

Reply:
Fixing manually.